recipes/coost/all/conanfile.py

```diff
@@ -0,0 +1,105 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir
+from conan.tools.microsoft import is_msvc, is_msvc_static_runtime
+import os
+
+required_conan_version = ">=1.50.0"
+
+
+class CoostConan(ConanFile):
+    name = "coost"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/idealvin/coost"
+    license = "MIT"
+    description = "A tiny boost library in C++11."
+    topics = ("coroutine", "cpp11", "boost")
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_libcurl": [True, False],
+        "with_openssl": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_libcurl": False,
+        "with_openssl": False,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            try:
+                del self.options.fPIC
+            except Exception:
+                pass
+
+    def requirements(self):
+        if self.options.with_libcurl:
+            self.requires("libcurl/7.80.0")
+        if self.options.with_libcurl or self.options.with_openssl:
+            self.requires("openssl/1.1.1q")
```

Comment:
```suggestion
            self.requires("openssl/1.1.1q")
        if self.settings.os == "Linux":
            self.requires("libbacktrace/cci.20210118")
```
Let's use backtrace library. It's used by Boost and Apache arrow